content/en/docs/guides/request-hedging.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Request Hedging"
+description : >-
+  Explains what request hedging is and how you can configure it.
+---
+
+### Overview
+
+Hedging is one of two configurable retry policies supported by gRPC. With
+hedging, a gRPC client sends multiple copies of the same request to different
+backends and uses the first response it receives. Subsequently, the client
+cancels any outstanding requests and forwards the response to the application.
+
+![Basic hedging diagram](../../../../static/img/basic_hedge.svg)
```

Comment:
The diagram doesn't seem to render, at least in the preview.

Reply:
Click on "Details" for the "netlify/grpc-io/deploy-preview" GitHub check and you will get a preview of the whole site with your change. The image does not show up there. Maybe a path issue?

Reply:
Renders now. Thanks. PTAL.